anomalib/data/base/dataset.py

```diff
@@ -21,7 +21,7 @@
 from anomalib.data.utils import read_image
 from anomalib.pre_processing import PreProcessor
 
-_EXPECTED_COLS_CLASSIFICATION = ["image_path", "label", "label_index", "split"]
+_EXPECTED_COLS_CLASSIFICATION = ["image_path", "split"]
```

Comment:
For video datasets, the samples dataframe contains a list of video files, which can't be assigned a single label. This is because each video file may consist of a combination of normal and anomalous frames.